account-kit/smart-contracts/src/ma-v2/account/common/modularAccountV2Base.ts

```diff
@@ -236,8 +237,29 @@ export async function createMAv2Base<
       }),
     );
 
-  const isAccountDeployed: () => Promise<boolean> = async () =>
-    !!(await client.getCode({ address: accountAddress }));
+  const isAccountDeployed: () => Promise<boolean> = async () => {
+    const code = await client.getCode({ address: accountAddress });
+    const is7702Delegated = code?.startsWith("0xef0100");
```

Comment:
I think RPCs usually return in lowercase, but just to be safe, let's lowercase before checking